### DISCUSSION TITLE
Delete photos on cloud

### DISCUSSION THREAD

**Discussion by UlfEichelquetscher** [opengisch/QField#4171] [Q&A]
Hey, I'm quite new in this topic, so sorry, if I'm using the wrong channel.
My problem is, when I add a photo to a new created point, the photo will remain in the cloud even if I delete the spatial created point. Is there any chance to delete photos in the cloud? It is not possible, rather through the website or the app.

Wish you all best!

**Comment by babakou9** [comment]
hello, 
you can delete the files locally and synchronize your project from qgis (with the 'prefer local' option selected). It will then delete the files from the cloud that aren’t present locally anymore.

hopefully this helped :)

**Reply by wanderingnature** [reply]
This is not necessarily the case.... I recently changed a base layer from raster GPKG to TIFF and removed the GPKG from project in QGIS and from local directory. I also deleted another GPKG vector layer from project at same time. When I synchronized the system did not remove the deleted local files from the cloud. They persisted and cannot be removed through web UI, I guess since they are the only version. The only way I could delete them was by using the qfieldcloud-sdk CLI.

**Reply by sheafile** [reply]
Hello, I'm having a similar issue as well. I've deleted all features in the QField app, and synchronized changed changes with my desktop. My Desktop DCIM folder is empty, but all historic photos are persisting in the Qfield app, and I have no option to delete the photos in the project folder. I'm running IOS. 

![IMG_1059](https://github.com/user-attachments/assets/a05cf30b-97b9-4ba6-a68f-2fdce1b4be0f)

**Comment by meyerlor** [comment]
@sheafile A solution for you would be to delete the project on your device (upload changed features first). Then redownload - if the pictures are not on the cloud anymore (because you deleted them on the desktop and pushed this deletion to the cloud) it will only sync the pictures which are in the cloud.

**Reply by meyerlor** [reply]
i ran in the same problem with PDFs created through QField - they are clogging the device space quite effectivly if you have an atlas with ~2000 features and a file for each one. I created an issue on the QField Ideas Platform - maybe you can upvote it there so the dev's might look into it.

https://ideas.qfield.org/qfieldcloud-feature-requests/p/better-pdf-handling-make-sync-deletion-possible

**Reply by sheafile** [reply]
Absolutely sir, appreciate you passing your fix along! Seems like a pretty notable issue that should be patched...

**Reply by nirvn** [reply]
Here's the solution: https://github.com/opengisch/QField/pull/5852 -- as of QField's next version (3.5), layouts will be written in the current project's subfolder named 'layouts', which will allow for synchronization with QFC (provided layouts is added to the attachments directories list) and will make it easier to clear by using the delete project folder from the local files view's 3-dot menu.

**Reply by meyerlor** [reply]
Sweet!
